### ISSUE TITLE
Options in the  ComboBox is not visible when mouse is hovered on them in high contrast white mode.

### ISSUE THREAD

**Issue by poonam0025** [OfficeDev/office-ui-fabric-react#11185]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: edge, windows 10.

Code pen link : https://codepen.io/poonam0025/pen/jOOewaM

Repro steps : 

1.  Go to link   : https://codepen.io/poonam0025/pen/jOOewaM
2.  Set system High Contrast mode : **High Contrast White**
3. Now click on combobox. Option in combobox are not visible here.

 

#### Actual behavior:

Option in combobox are not visible  in   **High Contrast White** mode.

#### Expected behavior:

Option in combobox should be visible  in   **High Contrast White** mode.

**Comment by poonam0025** [comment]
![FabricBug](https://user-images.githubusercontent.com/37584638/68835773-d12b3e80-06de-11ea-9783-6da46d0fb611.png)

**Comment by micahgodbolt** [comment]
yeah, easy to confirm this one. looks like the text colors need to be updated in high contrast 

![image](https://user-images.githubusercontent.com/1434956/68895492-392d5500-06de-11ea-8c1b-e28a91af9f74.png)
